frame/treasury/src/tests.rs

```diff
@@ -388,3 +388,18 @@ fn max_approvals_limited() {
 		);
 	});
 }
+
+#[test]
+fn remove_already_removed_approval_fails() {
+	new_test_ext().execute_with(|| {
+		Balances::make_free_balance_be(&Treasury::account_id(), 101);
+
+		assert_ok!(Treasury::propose_spend(Origin::signed(0), 100, 3));
+		assert_ok!(Treasury::approve_proposal(Origin::root(), 0));
```

Comment:
I am expecting to see some proof here that approval is placed, and a proof that it is removed after the next call. 

Checking the storage item directly should be enough.

Reply:
Thanks, I've updated it.